src/display/network.js

```diff
@@ -17,7 +17,6 @@ import {
   assert, createPromiseCapability, globalScope, isInt, MissingPDFException,
   UnexpectedResponseException
 } from '../shared/util';
-import { setPDFNetworkStreamClass } from './worker';
```

Comment:
The network.js must not be included in the firefox/mozcentral builds of pdf.js (as we did in pdf.worker.js) to reduce size. Notice that `gulp firefox`'s pdf.worker.js did not have any network.js code.